### ISSUE TITLE
Error at end of backup (1.07)

### ISSUE THREAD

**Issue by gintoddic** [pgbackrest/pgbackrest#255]
I see there was a backup completed in the info output, but I see this in the log.

`2016-09-09` 04:30:20.616 P00 ASSERT: [100]: hJob is required in BackupProcess->cmdResult
 at /usr/share/perl5/pgBackRest/Common/Log.pm line 265
                                        pgBackRest::Common::Log::logDebugProcess('pgBackRest::BackupProcess->cmdResult', '()', undef, 'ARRAY(0x2621810)', 'HASH(0x30e2a00)', 'HASH(0x3fdadc0)') called at /usr/share/perl5/pgBackRest/Common/Log.pm line 181
                                        pgBackRest::Common::Log::logDebugParam('pgBackRest::BackupProcess->cmdResult', 'ARRAY(0x2621810)', 'HASH(0x30e2a00)', 'HASH(0x3fdadc0)') called at /usr/share/perl5/pgBackRest/BackupProcess.pm line 50
                                        pgBackRest::BackupProcess::cmdResult(undef, 'pgBackRest::Protocol::LocalMaster=HASH(0x32e10d0)', undef) called at /usr/share/perl5/pgBackRest/Protocol/LocalProcess.pm line 289
                                        pgBackRest::Protocol::LocalProcess::process('pgBackRest::BackupProcess=HASH(0x2e0d088)') called at /usr/share/perl5/pgBackRest/Backup.pm line 411
                                        pgBackRest::Backup::processManifest(undef, 'pgBackRest::File=HASH(0x2676760)', '/dbdata/postgres/9.5-prod', '/dbdata/postgres/9.5-prod', 'full', 9.5, 1, 0, 'pgBackRest::Manifest=HASH(0x2676328)', ...) called at /usr/share/perl5/pgBackRest/Backup.pm line 857
                                        pgBackRest::Backup::process('pgBackRest::Backup=HASH(0x2594018)') called at /usr/bin/pgbackrest line 219
                                        eval {...} called at /usr/bin/pgbackrest line 252

                                   2016-09-09 04:30:20.650 P00   INFO: backup stop
2016-09-09 04:30:20.650 P00  ERROR: [198]: process terminated due to an unhandled exception:

2016-09-09 04:30:20.651 P00  ERROR: [141]: remote process terminated on local-1 host: ERROR [141]: unable to read line after 12000 seconds
2016-09-09 04:30:20.651 P00   WARN: unable to shutdown protocol [141]: remote process terminated on local-1 host: ERROR [141]: unable to read line after 12000 seconds
                                    HINT: the process completed successfully but protocol-timeout may need to be increased.`

**Comment by dwsteele** [comment]
I'll see what I can figure out, but it would be great if you turned on `log-level-file=debug` and provide some more context (~100 lines before the error) if you see this again.  Actually, the previous 100 lines from this log would be great - or the entire thing if you can.

I suspect the backup you are seeing in info is the previous backup - it definitely doesn't look like this one finished despite what the log is saying.  You can tell for sure if the backup is still in `[repo-path]/temp`.

**Comment by gintoddic** [comment]
I turned on the debug and started a new one. I attached the current log with the one that failed as well.
[data-backup.log.zip](https://github.com/pgbackrest/pgbackrest/files/465113/data-backup.log.zip)

**Comment by dwsteele** [comment]
The last line I see in the log is:
```
2016-09-10 00:17:01.188 P00  DEBUG:     Db::executeSql(): bIgnoreError = <false>, bResult = <true>, strSql = select to_char(current_timestamp, 'YYYY-MM-DD HH24:MI:SS.US TZ'), pg_xlogfile_name(lsn), lsn::text from pg_start_backup('pgBackRest backup started at 2016-09-10 00:16:55', false) as lsn
```
That's before all the good stuff happens...

**Comment by gintoddic** [comment]
Right I started a new one, there's a backup that finished yesterday in the log that has the error I pasted above.

**Comment by dwsteele** [comment]
OK, I'm starting to get a notion of what might be going on here.  Please send along the debug output when the new backup is done.

**Comment by gintoddic** [comment]
Backup seems to have completed. I see permissions errors in the debug of the log but access to the files seems fine. All files are cleared out of repo/temp directory.
[data-backup.log.zip](https://github.com/pgbackrest/pgbackrest/files/465940/data-backup.log.zip)

**Comment by gintoddic** [comment]
Here's same error failing with debug
[data-backup.log.zip](https://github.com/pgbackrest/pgbackrest/files/468376/data-backup.log.zip)

**Comment by dwsteele** [comment]
OK, I've found and fixed a few bugs that I'll be releasing in a day or so.  I want to verify against your latest log to make sure I got all of them.

**Comment by gintoddic** [comment]
great thank you.

**Comment by dwsteele** [comment]
Hopefully these issues are fixed by f54145c0ccd467.  v1.08 was just released with the fixes.

**Comment by dwsteele** [comment]
I'm going to close this.  If you see the same problems with 1.08 you can reopen the issue.
